clients/ts/signalr/spec/HubConnection.spec.ts

```diff
@@ -1,931 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
?

Reply:
Tests were moved from `spec` directory to `tests` directory, but this one was modified a bit, so it appears this way.